HW02_A_Graph-Fails.Rmd

```diff
@@ -29,20 +29,32 @@ library("magrittr") #so I can do some piping
 
 What error is being thrown? How do you correct it? (hint, the error message tells you)
 
+**My answer/comment:**

```

Comment:
Nice way of highlighting your answers and comments! I will try using it in my code.